content/millennium/r4.md

```diff
@@ -0,0 +1,477 @@
+---
+title: R4
+layout: api
+---
+
+# Overview
+
+This describes the resources that make up Cerner's implementation of the HL7<sup>®</sup> FHIR<sup>®</sup> standard. If you have any problems or requests,
+please post to our [developer group](https://groups.google.com/d/forum/cerner-fhir-developers).
+
+* TOC
```

Comment:
looking through it looks like our typical structure would be:
```
# R4

* TOC
{:toc}

## Overview
```
Is there a reason this is formatted how it is with everything nested under overview?

Reply:
this is the r4 counterpart to https://fhir.cerner.com/millennium/dstu2/